usr.sbin/daemon/daemon.c

```diff
@@ -33,6 +33,7 @@
 #include <sys/cdefs.h>
 __FBSDID("$FreeBSD$");
 
+#include <sys/event.h>
```

Comment:
Per style(9), `sys/param.h` or `sys/types.h` always come first, followed by the rest in alphabetical order.

Reply:
fixed